src/common/components/private-repos-info/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import url from 'url';
+import fetchJsonp from 'fetch-jsonp';
+
+import Popover from '../popover';
+import Button from '../vanilla/button';
+
+import styles from './private-repos-info.css';
+
+const MAILCHIMP_FORM_URL = 'https://canonical.us3.list-manage.com/subscribe/post-json';
+const MAILCHIMP_FORM_U = '56dac47c206ba0f58ec25f314';
+const MAILCHIMP_FORM_ID = '381f5c55f1';
+
+export default class PrivateReposInfo extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      showPopover: false,
+      popoverOffsetLeft: 0,
+      popoverOffsetTop: 0,
+      subscribeEmail: '',
+      subscribeSuccess: false,
+      subscribeError: false,
```

Comment:
To avoid creating actions/reducers for this fairly straightforward (and temporary) subscription form I just keep all the state in the component.